schema.sql

```diff
@@ -45,7 +43,40 @@ REFERENCES species(id);
 
 -- Add column owner_id which is a foreign key referencing the owners table
 ALTER TABLE animal
-ADD owner_id INT,
-CONSTRAINT owner_id
-FOREIGN KEY(owner_id)
+ADD owners_id INT,
+CONSTRAINT owners_id
+FOREIGN KEY(owners_id)
 REFERENCES species(id);
+
+
+CREATE TABLE dbo.vets
+(
+   id INT NOT NULL PRIMARY KEY, -- primary key column
+   name string NOT NULL,
+   age INT,
+   date_of_graduation DATE,
+   PRIMARY KEY(id)
+);
+GO
+
+CREATE TABLE dbo.specializations
+(
+   species_id INT,
+   vets_id INT,
+   CONSTRAINT fk_species FOREIGN KEY(species_id) REFERENCES species(id),
+   CONSTRAINT fk_vets FOREIGN KEY(vets_id) REFERENCES vets(id)
+);
+GO
+
+CREATE TABLE dbo.visits
+(
+   animal_id INT,
+   vets_id INT,

```

Comment:
- [ ] As suggested above please make the update the column name from `vets_id` to `vet_id`.

Note: Do not forget to also update the sections where you reference `vets_id`